packages/flutter/test/material/list_tile_test.dart

```diff
@@ -45,57 +45,65 @@ void main() {
   testWidgets('ListTile geometry (LTR)', (WidgetTester tester) async {
     // See https://material.io/guidelines/components/lists.html
 
+    final Key leadingKey = new GlobalKey();
+    final Key trailingKey = new GlobalKey();
     bool hasSubtitle;
 
-    Widget buildFrame({ bool dense: false, bool isTwoLine: false, bool isThreeLine: false }) {
+    Widget buildFrame({bool dense: false, bool isTwoLine: false, bool isThreeLine: false, double textScaleFactor: 1.0}) {
```

Comment:
flutter style has the spaces around named arguments

Reply:
Done.